docs/pages/workflow/debugging.md

```diff
@@ -28,11 +28,36 @@ Sometimes you'll be able to tell exactly what's wrong just by the [stacktrace](.
 
 If you are able to simplify your code as much as possible, tracking down the source of an error gets exponentially easier. That's exactly why so many open source repos require a [minimal reproducible demo](https://stackoverflow.com/help/minimal-reproducible-example) in their bug reports- it ensures you have isolated the issue and identified exactly where the problem lies! If your app is too large and complex to do that, try and extract the functionality you're trying to add to its own blank `npx create-expo-app` project, and go from there.
 
+### Native Debugging
+
+You can perform full native debugging with Xcode and Android Studio by generating the source code locally and building from source.
+
+#### Xcode
+
+> **Hint**: This is only available for macOS users.
+
+1. Generate the native code for your project with: `npx expo prebuild -p ios`
+   1. You can delete the `/ios` folder when you're done to ensure your project remains managed by Expo CLI. Keeping the folder and manually modifying it outside of `npx expo prebuild` means you'll need to manually upgrade and configure native libraries (Bare Workflow).
+2. Open the project in Xcode `xed ios`
+3. Build the app with <kbd>⌘r</kbd> (command R) or by pressing the play button in the upper left corner of Xcode.
```

Comment:
```suggestion
3. Build the app with <kbd>Cmd ⌘</kbd> + <kbd>R</kbd> or by pressing the play button in the upper left corner of Xcode.
```

This change references to our [writing style guide > Referencing keyboard shortcuts](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-keyboard-shortcuts).